src/PrettierEditProvider.ts

```diff
@@ -124,9 +125,10 @@ async function format(
     if (!hasConfig && vscodeConfig.requireConfig) {
         return text;
     }
-    const fileOptions = await resolveConfig(fileName, {
+
+    const fileOptions = (hasConfig ? await resolveConfig(fileName, {
```

Comment:
`hasConfig` means there is a prettier config (without editorconfig)
We still need to read editorconfig

In this case this means printing the error message twice a run.
I think you wrote this ternary to correct this 😄 

Our `resolveConfig` (the one you just wrote) could return
```js
{config: PrettierConfig|null, error?: Error}
```
and log only later.
I'm opened to better ideas!

Reply:
Yeah I was trying to work around the double-logging issue with the ternary, which is very hack-ish, but I was trying to be conservative about the changes I made :) I'll make the change you proposed and will updated the PR. Thanks!